projects/igniteui-angular/src/lib/grids/filtering/excel-style/excel-style-search.component.ts

```diff
@@ -46,6 +49,7 @@ export class IgxExcelStyleSearchComponent implements AfterViewInit {
     public ngAfterViewInit() {
         requestAnimationFrame(() => {
             this.virtDir.recalcUpdateSizes();
+            this.filteredData = null;
```

Comment:
Do we need to set it to null here? Is it possible to have an old value?

Reply:
Not for the time being, so we'll remove it.